### ISSUE TITLE
Compass alpha 8 generates too many prefixes

### ISSUE THREAD

**Issue by mgol** [Compass/compass#1417]
This code:

``` scss
@import "compass/css3";

// It seems messing with these numbers doesn't change anything:
$graceful-usage-threshold: 1;
$critical-usage-threshold: 1;

$supported-browsers: ("chrome", "firefox", "ie", "safari");

$browser-minimum-versions: (
    "chrome": "30",
    "firefox": "24",
    "ie": "10",
    "safari": "7"
);

div {
    @include transform(scaleX(2));
}
```

produces this output with Compass 0.13.alpha.8:

``` css
div {
  -moz-transform: scaleX(2);
  -ms-transform: scaleX(2);
  -webkit-transform: scaleX(2);
  transform: scaleX(2);
}
```

even though only a WebKit prefix is needed.

A configuration `config.rb` file used:

``` ruby
css_dir = "."
sass_dir = "."
output_style = :expanded
line_comments = false
```

**Comment by cimmanon** [comment]
Why do you believe that code will do anything?  None of those variables match anything in the Compass source code.

**Comment by mgol** [comment]
@cimmanon That's not true. See e.g. https://github.com/chriseppstein/compass/blob/master/frameworks/compass/stylesheets/compass/_support.scss#L20-26

Also, this approach is even documented. http://beta.compass-style.org/help/documentation/tuning-vendor-prefixes/

**Comment by chriseppstein** [comment]
A few things wrong here:
1. Several features thresholds were not defaulted correctly. This did not affect your code though. But I did discover it and fixed it as 2e34dd390eb81838be86e6f46b5ece941d9c010d.
2. Config variables have to be set before importing. See http://compass-style.org/help/tutorials/configurable-variables/ for details.
3. Prefix support is based on an **OR** of threshold and minimums. Setting your minimums to 1% means that several other browsers are still needed even though your minimums are higher. To disable the thresholds and rely only on your minimums you'd need to set them to `100`, not `1`. I'll make the docs a bit more clear on this.

So with this code:

``` scss
$graceful-usage-threshold: 100;
$critical-usage-threshold: 100;

@import "compass/css3";
$debug-browser-support: true;

$supported-browsers: ("chrome", "firefox", "ie", "safari");

$browser-minimum-versions: (
    "chrome": "30",
    "firefox": "24",
    "ie": "10",
    "safari": "7"
);

div {
    @include transform(scaleX(2));
}
```

I get the following output:

``` css
div {
  -webkit-transform: scaleX(2);
  transform: scaleX(2); }
```

Also, if you turn on debugging you'll see a lot of details about Compass's decision making process. Here's why at 1% you still need prefixes:

``` css
div {
  /* Capability transforms2d is prefixed with -moz because 1.43011% of users need it which is more than the threshold of 1%. */
  /* Creating new -moz context. */
  -moz-transform: scaleX(2);
  /* Capability transforms2d is prefixed with -ms because 4.94898% of users need it which is more than the threshold of 1%. */
  /* Creating new -ms context. */
  -ms-transform: scaleX(2);
  /* Capability transforms2d is prefixed with -webkit because 51.43012% of users need it which is more than the threshold of 1%. */
  /* Creating new -webkit context. */
  -webkit-transform: scaleX(2);
  transform: scaleX(2); }
```

**Comment by mgol** [comment]
@chriseppstein The debugging feature is extremely cool, I'll now know how to debug those kind of issues myself. Thanks for the help, great work with the new Compass!
